lib/_http_agent.js

```diff
@@ -116,6 +117,24 @@ function Agent(options) {
       }
     }
   });
+
+  // Don't emit keylog events unless there is a listener for them.
+  const enableKeylog = (eventName) => {
```

Comment:
Can't this be moved out into the parent scope for reuse?

Reply:
yes, PTAL